### PR TITLE
[REL-891063] - Code migration to GitHub-MassImport - main

### PR DESCRIPTION
**Pull request by sameerrelone**
- Cloned repo from BitBucket to GitHub, updated changes as per SOP and created pipeline in AzDo.
- Updated changelog and removed Tridentfile and trident scripts
- Server dev verification document [REL-891063 Server Dev Verification-SLFB.docx](https://github.com/relativityone/server-datatransfer-relativity-data-transfer-legacy-sdk/files/13683699/REL-891063.Server.Dev.Verification-SLFB.docx)

### CODE REVIEW

File: build.ps1
Comment (ravikonnur) [inline build.ps1:1]: Replace Write-Progress function with Write-Host in all the places.

File: build.ps1 [hunk 1]
```diff
@@ -73,7 +74,19 @@ Write-Progress "Importing required Powershell modules..."
...
 Import-Module (Join-Path $ToolsDir "kCura.PSBuildTools.*\PSBuildTools.psd1") -ErrorAction Stop
-Install-Module VSSetup -Scope CurrentUser -Force
+
+if (!(Get-Module -Name VSSetup -ListAvailable))
+{
+    Install-Module VSSetup -Scope CurrentUser -Repository 'powershell-anthology' -Force
+}
+Import-Module VSSetup -Force
+
+$ReportGenerator = Join-Path $ToolsDir "reportgenerator.exe"
```
Comment (ravikonnur) [inline build.ps1:84]: Remove this block of code as it is not necessary.

**Comment by ravikonnur** [comment]
@sameerrelone , Remove Tridentfile and related scripts.

File: psakefile.ps1 [hunk 2]
```diff
@@ -127,11 +127,9 @@ function Invoke-Tests
     if($WithCoverage)
     {
         $OpenCover = Join-Path $BuildToolsDir "opencover.*\tools\OpenCover.Console.exe"
-        $ReportGenerator = Join-Path $BuildToolsDir "reportgenerator.*\tools\net47\ReportGenerator.exe"
```
Comment (ravikonnur) [inline psakefile.ps1:130]: Removing this line will fix any build error or other reason?

File: psakefile.ps1 [hunk 3]
```diff
@@ -127,11 +127,9 @@ function Invoke-Tests
...
         $CoveragePath = Join-Path $LogsDir "Coverage.xml"
 
         exec { & $OpenCover -target:$NUnit -targetargs:"$Solution --where=\`"$WhereClause\`" --noheader --labels=On --skipnontestassemblies --result=$OutputFile $settings" -register:user -filter:"+[*DataTransfer.Legacy*]* -[*Tests*]* -[*NUnit*]*" -hideskipped:All -output:"$LogsDir\OpenCover.xml" -returntargetcode }
-        exec { & $ReportGenerator -reports:"$LogsDir\OpenCover.xml" -targetdir:$LogsDir -reporttypes:Cobertura }
```
Comment (ravikonnur) [inline psakefile.ps1:134]: Removing this line will fix any build error or other reason?

**Review by ravikonnur — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -27,5 +27,7 @@ For more information on RAP CD, [view the documentation in Einstein](https://ein
 
 ## History
 
-Tag: 0.2.3
-Branch: server-main
\ No newline at end of file
+The server-main branch was migrated from https://git.kcura.com/projects/DTX/repos/relativity-data-transfer-legacy-sdk/browse/version.txt?at=refs%2Fheads%2Fserver-main by the Server Data Transfer Team. Tag: 24000.0.1-server Branch: server-main
+
+## Maintainers
+This repository is owned by the Server Data Transfer Team. #help-server-data-transfer
```
[the remark this answers is not in the document]
Comment (shivarajkamadod) [reply README.md:33]: image

File: buildtools/packages.config [hunk 5]
```diff
@@ -7,6 +7,5 @@
 	<package id="NUnit.Extension.TestExecutionParser" version="0.2.2" />
 	<package id="psake-rel" version="5.0.0" />
 	<package id="opencover" version="4.7.1221" />
-	<package id="reportgenerator" version="4.1.9" />
```
[the remark this answers is not in the document]
Comment (sameerrelone) [reply buildtools/packages.config]: It was not in the SOP, after removing this i got compile error then i re-added this line then got fixed error

**Review by shivarajkamadod — APPROVED** [review]

**Review by ravikonnur — APPROVED** [review]

**Review by shivarajkamadod — APPROVED** [review]

**Pull request merged**